api/policies/hasValidMoveBody.js

```diff
@@ -27,6 +27,20 @@ module.exports = function (req, res, next) {
       }
     }
       return next();
+
+    case MoveType.SCUTTLE: {
+      if (!cardId && !targetId) {
```

Comment:
We need both so if either is missing it's an error
```suggestion
      if (!cardId || !targetId) {
```